docs/user_endpoints.md

```diff
@@ -217,3 +218,28 @@ Status: 200
   ]
 }
 ```
+
+## GET /users/members/me
+Request
+```ruby
+  GET /users/members/me
+```
+
+Response
+```ruby
+Status: 200
+{
+  "id": 1,
+  "name": "Firstname Lastname",
+  "points": 10,
+  "image": "<http://i.imgur.com/GZIxExK.png>",
+  "url": "<http://domain.com/users/1.json>",
```

Comment:
nice, you don't need to include the outermost brackets, it could make someone think he first has to remove them before using